bofire/plot/duplicates.py

```diff
@@ -0,0 +1,70 @@
+from typing import List
+
+import numpy as np
+import pandas as pd
+import plotly.graph_objects as go
+
+
+def plot_duplicates_plotly(
+    experiments: pd.DataFrame, duplicates: List[str], key: str, scale: float = 0.1
+) -> go.Figure:
+    """
+    Plots duplicates using Plotly.
+
+    Args:
+        experiments: The dataframe containing the experiments data.
+        duplicates: A list of strings representing the labcodes of the duplicates.
```

Comment:
What are labcodes?

Reply:
Hi @bertiqwerty ,

these is an optional column in the `experiments` data frame which can be a string based unique identifier of one experiment.

Best,

Johannes